### PR TITLE
smoketestbase-vxRetainReferenceBase fix

### PR DESCRIPTION
**Pull request by LakshmiKumar23**

### CODE REVIEW

File: amd_openvx/openvx/ago/ago_interface.cpp [hunk 1]
```diff
@@ -1406,7 +1409,7 @@ vx_status agoVerifyNode(AgoNode * node)
 					}
 					// re-initialize, if updated
 					if (updated) {
-						char scale[256], desc[512];
```
Comment (kiritigowda) [inline amd_openvx/openvx/ago/ago_interface.cpp]: any reason to reduce the size?

Comment (LakshmiKumar23) [reply amd_openvx/openvx/ago/ago_interface.cpp]: I thought that was from the changes one of you commited. I did a merge with your branch before pushing my changes

Comment (LakshmiKumar23) [reply amd_openvx/openvx/ago/ago_interface.cpp]: Changed it to what it was. The rest of the changes are mine

Comment (kiritigowda) [reply amd_openvx/openvx/ago/ago_interface.cpp]: LGTM

**Review by kiritigowda — APPROVED** [review]

**Pull request merged**